### ISSUE TITLE
Debian packages should have distro tag in filenames

### ISSUE THREAD

**Issue by rtsisyk** [packpack/packpack#11]
For example, `libmsgpuck-dev_1.0.3.3-1_amd64.deb` should be `libmsgpuck-dev_1.0.3.3-1~stretch_amd64.deb`.

Needed for multiple builds in the same directory and for `mkrepo` tool.

**Comment by gdevenyi** [comment]
Since I have to build locally this is a priority for me, I'll see if I can figure it out.

**Comment by rtsisyk** [comment]
Yes, because all Debian/Ubuntu packages have the same filename. I found
that distribution name can be added as a part of package version during
"dch" invocation. Currently I'm trying to find any specification to choose
between 1.2.3-wheezy and 1.2.3~wheezy conventions.

On Nov 25, 2016 21:54, "Gabriel A. Devenyi" <notifications@github.com>
wrote:
>
> Since I have to build locally this is a priority for me, I'll see if I
can figure it out.

**Comment by rtsisyk** [comment]
BTW, I also struggled to get the name of distribution from dpkg.

On Nov 25, 2016 21:54, "Gabriel A. Devenyi" <notifications@github.com>
wrote:
>
> Since I have to build locally this is a priority for me, I'll see if I
can figure it out.
